catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/intents/looks/paintroid/PocketPaintNewSpriteDiscardIntentTest.java

```diff
@@ -107,9 +114,48 @@ public void testCancelImport() {
 
 		intended(expectedIntent);
 
+		closeSoftKeyboard();
+
+		onView(withText(newSpriteName))
+				.check(matches(isDisplayed()));
+
+		onView(withText(R.string.new_empty_sprite_note))
+				.check(matches(isDisplayed()));
+
+		onView(withText(R.string.cancel))
+				.perform(click());
+
 		onRecyclerView().checkHasNumberOfItems(1);
 	}
 
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void testAddEmptySprite() {
+		onView(withId(R.id.button_add))
+				.perform(click());
+
+		onView(withId(R.id.dialog_new_look_paintroid))
+				.perform(click());
+
+		intended(expectedIntent);
+
+		closeSoftKeyboard();
+
+		onView(withText(newSpriteName))
+				.check(matches(isDisplayed()));
+
+		onView(withText(R.string.new_empty_sprite_note))
+				.check(matches(isDisplayed()));
+
+		onView(Matchers.allOf(withId(android.R.id.button1), withText(R.string.ok)))
+				.perform(click());
```

Comment:
Can you please make a method for the whole code above (sth. like `addEmptySprite()`) to avoid code duplication in both testcases.